.github/workflows/docker.yaml

```
@@ -0,0 +1,112 @@
+#
+# GitHub actions for building and testing.
+#
+# For best support, use `-latest` for runners spinning up containers. More at
+# https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners.
+
+name: Docker
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+# Set to 'yes' to open a tunnel to GitHub's VMs through tmate on failures.
+# Also increase timeout-minutes for the relevant OS when debugging remotely.
+env:
+  CHEVAH_DEBUG: 'no'
+
+# Using a job name that doesn't contain the OS name, to minimize the risk of
+# confusion with the OS names of the containers, which are the relevant ones.
+jobs:
+  latest:
+    runs-on: ubuntu-latest
```

latest is a very generic name for a job... and I am not sure what the job does....

I guess that this is not a big issue with Pythia as the purpose of any job is to build test and publish.

```suggestion
  build_test_and_publish:
```

---

You are right in principle. But the name was shortened to make the list of build jobs more easy to parse by us, humans. Take a look at https://github.com/chevah/pythia/runs/2361586638 with both _Docker_ and _Bare_ jobs visible.

There is also a comment above that tries to explain this: _# Using a job name that doesn't contain the OS name, to minimize the risk of confusion with the OS names of the containers, which are the relevant ones._